packages/actions/index.tsx

```diff
@@ -6,10 +6,15 @@ import s from './style.module.css'
 export default function Actions({
   appearance = 'light',
   layout = 'inline',
-  brand = 'hashicorp',
+  theme = 'hashicorp',
```

Comment:
I think there should be a **major** changeset for `@hashicorp/react-actions` to account for this prop change

Reply:
@thiskevinwang agreed, and I can make that update. Main reason to do minor for now was because this component was not being consumed anywhere just yet.